app/assets/src/components/pipeline_sample_reads.scss

```diff
@@ -9,69 +9,33 @@
     margin-bottom: 10px;
   }
 
-  .topRow {
-    display: flex;
+  .viewHeader {
     margin-top: 20px;
```

Comment:
Seems to me this distances should be standard, no ?

Reply:
The Samples Report Header has the "pipeline" text above it, and the tabs below it, while the Phylo Tree one doesn't, so they require different distances. 

We can definitely standardize it in the future if it turns out there's only two different cases, but it seems a little premature now with only two headers.